cmd/ovnk-mcp-server/main.go

```diff
@@ -26,12 +27,16 @@ type MCPServerConfig struct {
 func main() {
 	serverCfg := parseFlags()
 
+	if serverCfg.Mode != "both" && serverCfg.Mode != "offline" && serverCfg.Mode != "live-cluster" {
+		log.Fatalf("Invalid mode: %s. Valid modes are: live-cluster, offline, both", serverCfg.Mode)
+	}
+
 	ovnkMcpServer := mcp.NewServer(
 		&mcp.Implementation{Name: "ovn-kubernetes"},
 		&mcp.ServerOptions{HasTools: true},
 	)
 
-	if serverCfg.Mode == "live-cluster" {
+	if serverCfg.Mode == "live-cluster" || serverCfg.Mode == "both" {
```

Comment:
can have two functions like `setupLiveCluster` and `setupOffline`, then invoke it with switch case ?
something like below:
```
switch serverCfg.Mode {
case "live-cluster":
	setupLiveCluster(ovnkMcpServer, serverCfg)

case "offline":
	setupOffline(ovnkMcpServer)

case "both":
	setupLiveCluster(ovnkMcpServer, serverCfg)
	setupOffline(ovnkMcpServer)

default:
	log.Fatalf("Invalid mode: %s", serverCfg.Mode)
}
```